README.md

```diff
@@ -71,6 +71,7 @@ From ourselves and the community!
 | [Perl 5](https://github.com/bk/Data-ULID) | [bk](https://github.com/bk) | ✓ |
 | [PHP](https://github.com/Lewiscowles1986/ulid) | [Lewiscowles1986](https://github.com/Lewiscowles1986) |
 | [PHP](https://github.com/robinvdvleuten/php-ulid) | [robinvdvleuten](https://github.com/robinvdvleuten) |
+| [PHP](https://github.com/symfony/uid) | [Symfony](https://github.com/symfony) |
```

Comment:
The Symfony UID component contains binary format implementation, I believe.

```suggestion
| [PHP](https://github.com/symfony/uid) | [Symfony](https://github.com/symfony) | ✓ |
```

Reply:
Thanks, updated!